### DISCUSSION TITLE
-manifest to accept an array of manifest ids

### DISCUSSION THREAD

**Discussion by FatalBulletHit** [SteamRE/DepotDownloader#218] [Q&A]
This could be implemented by having 2 parameters sets, so you can either pass an array of depot ids or an array of manifest ids in combination with a single depot id, but not both.

**Comment by azuisleet** [comment]
This is not feasible to implement as we expect only one version of each depot.

**Reply by FatalBulletHit** [reply]
But why do we only expect one version? If only one depot id is specified, there shouldn't be any issues with multiple manifest ids, no?
